### DISCUSSION TITLE
Automatically booting to a selected device

### DISCUSSION THREAD

**Discussion by Zero3K** [Genymobile/feedback#25] [Genymotion Desktop] [answered]
It would be nice if it had an option in a device's option to have it automatically boot it when starting Genymotion Desktop. When doing so, the main dialog (with the list of devices) should not be shown.

**Comment by Zero3K** [comment]
Well?

**Reply by agavignet** [reply, marked as answer]
@Zero3K Thank you for your suggestion, the product team will think about a better integration but meanwhile you can call the emulator directly with a virtual device name. For example if you have a virtual device named "Zero3K", you can start `./player -n Zero3K`. Or `.\player.exe -n Zero3K` if you're on Windows. You could also create a dedicated shortcut for that device.
